connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -99,7 +99,6 @@ private class UCProxy extends TableCatalog {
       tablesApi.getTable(name + "." + ident.toString)
     } catch {
       case e: ApiException if e.getCode == 404 =>
-        e.printStackTrace()
```

Comment:
@cloud-fan in your follow up PR can you set up some slf4j logging?

Reply:
sure